test/runtime/ferguson/testatomics.test.c

```diff
@@ -74,6 +74,8 @@ int main(int argc, char** argv)
 {
   atomic_bool flag;
 
+  memset(&flag, 0, sizeof(flag));
```

Comment:
Is it important to you that we test it? This whole business came from moving an old test that wasn't getting run at all... and I'm not that interested in making the test better as much as just getting it out of the include/ directory.

Anyway, I didn't remember there was such a thing when I was looking at the valgrind error.